Flexispot.py

```diff
@@ -5,7 +5,7 @@
 from machine import UART, Pin
 from umqttsimple import MQTTClient
 
-class ControlPanel:
+class Controlpanel:
```

Comment:
I wouldn't rename it like that since the python style guide suggests PascalCase for class names